### DISCUSSION TITLE
Cursor pagination

### DISCUSSION THREAD

**Discussion by ithuis** [saloonphp/saloon#393] [Q&A]
I am working on a project that consumes a cusorPagination. This paginator works if i use it in 1 function, like in the example. 
But i want to load the results per page on a livewire page and add a button "go to next page".

**Comment by leo95batista** [comment]
@ithuis  I'm working in something similar to.. when I use the example in the docs, the generator just iterates all the API pages instead one by one, do you figure out how to make it work?

**Comment by wivaku** [comment]
This worked for me:
```
$spotifyConnector = new SpotifyConnector;
$paginator = $spotifyConnector->paginate($request);

$response = $paginator->current();
$items = $response->json();

$paginator->next();

$response = $paginator->current();
$items = $response->json();
```
